spec/models/trip_spec.rb

```diff
@@ -15,5 +15,6 @@
 
   describe 'relationships' do
     it{should respond_to :stations}
+    it{is_expected.to belong_to(:condition)}
```

Comment:
Clee has committed to exploring a way to make this relationship work without seeding an condition_index field.